yaml/src/main/java/com/fasterxml/jackson/dataformat/yaml/YAMLFactoryBuilder.java

```diff
@@ -117,6 +126,12 @@ public YAMLFactoryBuilder stringQuotingChecker(StringQuotingChecker sqc) {
         return this;
     }
 
+
+    public YAMLFactoryBuilder nodeStyleResolver(NodeStyleResolver nodeStyleResolver) {
```

Comment:
Could you add `@since 2.13` javadoc to this (and getter) too?

Reply:
done :)